util/testutil/integration/dockerd.go

```diff
@@ -63,6 +64,18 @@ func (c moby) New(ctx context.Context, cfg *BackendConfig) (b Backend, cl func()
 		return nil, nil, err
 	}
 
+	bkcfg, err := config.LoadFile(cfg.ConfigFile)
+	if err != nil {
+		return nil, nil, errors.Wrapf(err, "failed to load buildkit config file %s", cfg.ConfigFile)
+	}
+
+	var mirrors []string
```

Comment:
could we set up the entitlements as well

Reply:
I see we set:

https://github.com/moby/buildkit/blob/234dfca4a2fb52f143e66a9fd962b466f209af9f/client/client_test.go#L8582-L8584

https://github.com/moby/buildkit/blob/234dfca4a2fb52f143e66a9fd962b466f209af9f/client/client_test.go#L8591-L8593

I guess it's just for containerd snapshotter right?:

https://github.com/moby/moby/blob/214cd9d1e647373a27991f299a1bf05b758df05f/daemon/config/builder.go#L63-L67